Assertion redeclared during import w/ Gomega

```golang
package foo

import (
    "testing"
    . "github.com/onsi/gomega"
    . "github.com/franela/goblin"
)
```

trying to run `go test` produces the following error:

```
./foo_test.go:6:5: Assertion redeclared during import "github.com/franela/goblin"
        previous declaration during import "github.com/onsi/gomega"
FAIL    foo/foo [build failed]
```

go.mod
```golang
module foo/foo

go 1.14

require (
	github.com/franela/goblin v0.0.0-20200611003024-99f9a98191cf
	github.com/onsi/gomega v1.10.1
        ...
)
```

--- a/README.md
+++ b/README.md
@@ -111,12 +111,12 @@ package foobar
 
 import (
     "testing"
-    . "github.com/franela/goblin"
+    goblin "github.com/franela/goblin"
     . "github.com/onsi/gomega"
 )
 
 func Test(t *testing.T) {
-    g := Goblin(t)
+    g := goblin.Goblin(t)
 
     //special hook for gomega
     RegisterFailHandler(func(m string, _ ...int) { g.Fail(m) })

